cfme/tests/automate/test_instance.py

```diff
@@ -219,3 +219,59 @@ def test_automate_relationship_trailing_spaces(request, klass, namespace, domain
             assert result.output == ""
         else:
             assert search in result.output
+
+
+@pytest.fixture(scope="module")
+def copy_instance(domain):
+    """
+    This fixture copies the instance '/ManageIQ/System/Request/ansible_tower_job' to new domain.
+    """
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
```

Comment:
Optional: That's a lot of instantiations! Splitting this up into (at least 2) steps would improve readability.